viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    
+    if title and genre and rating:
+        new_movie = {}
+        new_movie["title"] = title
+        new_movie["genre"] = genre
+        new_movie["rating"] = rating
+        return new_movie
+    else:
+        return None

```

Comment:
I'd suggest flipping the order of the conditions around to handle the special case (ie, checking if something is falsy) first, so that the code can quickly check and get out of there. This arrangement is referred to as a "guard clause" or a "guard condition." Notice that it lets us move the main focus of the function (creating a dictionary from the parameters) out of an indented block, letting it stand out more clearly.

```suggestion
    if not title or not genre or not rating:
        return None
    new_movie = {}
    new_movie["title"] = title
    new_movie["genre"] = genre
    new_movie["rating"] = rating
    
    return new_movie
```

We could also create the dictionary with all keys defined at once to dry up your code a little, but in this case it makes a miniscule difference